src/test/java/io/ticktok/TicktokBrokenServerTest.java

```diff
@@ -0,0 +1,38 @@
+package io.ticktok;
+
+import io.ticktok.support.TicktockServiceStub;
+import org.junit.After;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static io.ticktok.support.TicktockServiceStub.TICKTOK_SERVICE_DOMAIN;
+import static io.ticktok.support.TicktockServiceStub.TOKEN;
+
+public class TicktokBrokenServerTest {
```

Comment:
to make it relate to its class I would suggest: TicktokTest_BrokenServer

Reply:
Fixed